qa/tasks/cephfs/cephfs_test_case.py

```diff
@@ -199,7 +201,13 @@ def setUp(self):
 
         self.configs_set = set()
 
+        log.info('Finished running CephFSTestCase.setUp().')
+        self.run_ceph_cmd('fs dump')
+
     def tearDown(self):
+        log.info('Running CephFSTestCase.tearDown() now.')
+        self.run_ceph_cmd('fs dump')
```

Comment:
are these pretty-printed?

Reply:
It'd also be nice to print the mon leader as that's usually the log we need to grep.

Reply:
Added.